### PR TITLE
optimize nacos config client code

### PR DESCRIPTION
**Pull request by mengxiangrui007**

### CODE REVIEW

**Comment by flystar32** [comment]
@mengxiangrui007 
how about provide your DingDing account (if you don't have a DingDing account, Wechat is ok), We want to invite you to SCA developers' DingDing group.

File: spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/refresh/NacosContextRefresher.java [hunk 1]
```diff
@@ -108,11 +119,16 @@ public void receiveConfigInfo(String configInfo) {
...
 					}
 					catch (NoSuchAlgorithmException | UnsupportedEncodingException e) {
-						logger.warn("[Nacos] unable to get md5 for dataId: " + dataId, e);
+						LOGGER.warn("[Nacos] unable to get md5 for dataId: " + dataId, e);
 					}
 				}
 				refreshHistory.add(dataId, md5);
-				contextRefresher.refresh();
+				applicationContext.publishEvent(
+						new RefreshEvent(this, null, "Refresh  Nacos config"));
```
Comment (fangjian0423) [inline spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/refresh/NacosContextRefresher.java]: duplicate blank

File: spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/refresh/NacosContextRefresher.java [hunk 2]
```diff
@@ -108,11 +119,16 @@ public void receiveConfigInfo(String configInfo) {
...
+				if (LOGGER.isDebugEnabled()) {
+					LOGGER.debug("Refresh  Nacos config group[[]] dataId[]", group,
```
Comment (fangjian0423) [inline spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/refresh/NacosContextRefresher.java]: duplicate blank

**Comment by mengxiangrui007** [comment]
> @mengxiangrui007
> how about provide your DingDing account (if you don't have a DingDing account, Wechat is ok), We want to invite you to SCA developers' DingDing group.


@flystar32 Thank you for your invitation,My Wechat account is WHMXY_.

File: spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/refresh/NacosContextRefresher.java (hunk 1 above)
Comment (mengxiangrui007) [reply spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/refresh/NacosContextRefresher.java]: Thank you for your review,I have fix it .

**Pull request merged**